### PR TITLE
Add Byte Validator method to check valid byte array

### PR DESCRIPTION
**Pull request by palayusuf**
byte-validator-method

### CODE REVIEW

File: src/main/java/org/example/ByteValidator.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.example;
+
+public class ByteValidator {
+    public static boolean isValidByteArray(byte[] byteArray, String validCharacters) {
```
Comment (palaevren) [inline src/main/java/org/example/ByteValidator.java:4]: Do not create static method

Comment (palaevren) [inline src/main/java/org/example/ByteValidator.java:4]: The second argument must not be a string of valid characters. It must be the name of **Character set**

I want to test if the byte array is a valid string in that character set. Check out `CharacterSet` class in java:

File: src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -20,5 +20,10 @@ public static void main(String[] args) {
         stringexample str = new stringexample();
         String message = "Hi, today is a very nice day.";
         str.printMessageDetails(message);
-    }
+        ByteValidator bv = new ByteValidator();
+        byte[] byteArray ="Hello".getBytes();
+        String validCharacters = "Hello";
+        ByteValidator bv1 = bv;
+        bv1.isValidByteArray(byteArray, validCharacters);
```
Comment (palaevren) [inline src/main/java/org/example/Main.java:27]: We don't want to test our code in a main method. We should use unit test class only

Comment (palayusuf) [reply src/main/java/org/example/Main.java]: I wrote this here not to test it in main but to see how it works in main.

**Pull request merged**